app/Http/Controllers/api/v1/AdminAuthController.php

```diff
@@ -37,6 +37,13 @@ public function login(Request $request)
 
         $passport_client = Passport::client()->where('name', "admins")->first();
 
+        if (empty($passport_client)){
```

Comment:
i think here we should provide a server error to the client 
```php
abort_unless($passport_client, 500, 'authentication server error')
```
reason is that the client would be confused about his credentials

Reply:
Yes, The abort_unless helper functions allow for the writing of code that reads easier and takes up less space